src/components/AnonymousChat/child/ChatReplyPreview.tsx

```diff
@@ -0,0 +1,111 @@
+/* eslint-disable no-use-before-define */
+import React from 'react';
+import Animated, {SlideInDown, SlideOutDown} from 'react-native-reanimated';
+import {StyleSheet, Text, TouchableOpacity, View} from 'react-native';
+
+import IconClear from '../../../assets/icon/IconClear';
+import dimen from '../../../utils/dimen';
+import useMessageHook from '../../../hooks/screen/useMessageHook';
+import {colors} from '../../../utils/colors';
+import {fonts, normalizeFontSize} from '../../../utils/fonts';
+
+interface ChatReplyPreviewProps {
+  type: 'SIGNED' | 'ANONYMOUS';
+}
+
+const ChatReplyPreview = ({type}: ChatReplyPreviewProps) => {
+  const {replyPreview, clearReplyPreview} = useMessageHook();
+
+  const textContainerStyle = [
+    type === 'SIGNED' ? styles.containerSigned : styles.containerAnon,
```

Comment:
Better we using the SIGNED constant variable, instead of using string.
This is for SIGNED and ANONYMOUS

Reply:
solved in new commit